io/disk/dbench.py

```diff
@@ -89,24 +98,36 @@ def setUp(self):
         process.run('patch -p1 < %s' % self.get_data(patch), shell=True)
         process.run('./configure')
         build.make(self.sourcedir)
-        self.dirs = self.disk
         if self.disk is not None:
             if self.disk in disk.get_disks():
                 if raid_needed:
-                    raid_name = '/dev/md/mdsraid'
+                    raid_name = '/dev/%s' % self.md_name
                     self.create_raid(self.disk, raid_name)
                     self.raid_create = True
                     self.disk = raid_name
-                    self.dirs = self.disk
                 if lv_needed:
                     self.disk = self.create_lv(self.disk)
                     self.lv_create = True
-                    self.dirs = self.disk
                 if fstype:
-                    self.dirs = self.workdir
-                    self.create_fs(self.disk, self.dirs, fstype)
+                    self.create_fs(self.disk, fstype)
                     self.fs_create = True
 
+    def pre_cleanup(self):
+        umount_dir = "umount -f %s" % self.mountpoint
```

Comment:
can we use self.disk_obj.umount()? or self.disk_obj.unmount(force=True) instead direct command.

Reply:
I had the similar thought, but did not work, as the disk under test varies in mux variants..  disk_obj is /dev/mapper/mapth  or it can be logical device.. or raid mapper device... so instead of handling differnt disk obj ..easy solution is to umount /mnt i.e mount irespective of disk

Also before the test start we are not knowing the disk obj, or lv object or raid obj.. so is the reason I did not use the libraries... this is only pure pre cleanup.. unlike teardown where we have initiated objs

Reply:
ok, sure. I am planing to develop a util funtion which will take care of the cleaning part even before running the test.
for now, since it is blocking CI. we can merge this.